src/main/java/seedu/expenditure/Expenditure.java

```diff
@@ -2,7 +2,7 @@
 
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
-import static seedu.expenditure.CurrencyValue.sgDConversion;
+import static seedu.expenditure.CurrencyValue.sgdConversion;
```

Comment:
Nice catch! However, we should keep package names to all lower case as per convention